lib/xmerl/doc/src/xmerl_ug.xmlsrc

```diff
@@ -359,6 +359,21 @@ Data =
     ...    </pre>
     <p>The result will be: </p>
     <codeinclude file="new_motorcycles2.txt" tag="" type="none"></codeinclude>
+    <p>
+
+    The generated XML above was formatted for readability.
+
+    Another exporter which indents the code with 2 spaces can also be used.
+
+    This exporter should make the xml a bit more human-readable without the need
+    to use external tools such as xmllint.
```

Comment:
IMO, this is too opinionated as to what constitutes human-readable. The paragraph should instead merely mention that there exists another exporter and what its _intention_ is (vs "it should produce this effect on a reader"). I would also leave out the "a bit" part (how much is "a bit"?).

Reply:
"This exporter formats the xml in the same manner as xmllint." <- Better alternative / Or simply remove lines 368-369.

I could not find a super simple - or - elegant manner to allow a user to use another amount of spaces, as for instance xmllint provides the option using XMLLINT_INDENT environment variable.

Really appreciate your feedback! ;)